ui/src/containers/ClusterInfoPage/DownloadArtifactsModal.tsx

```diff
@@ -15,16 +15,13 @@ type ArtifactsListProps = {
 
 function ArtifactsList({ artifacts }: ArtifactsListProps): ReactElement {
   return (
-    <ul className="list-disc ml-5 pb-5">
+    <List className="pf-u-mb-md">
       {artifacts.map((artifact) => (
-        <li key={artifact.Name}>
-          <a href={artifact.URL} className="underline text-blue-500">
-            {artifact.Name}
-          </a>{' '}
-          - {artifact.Description}
-        </li>
+        <ListItem key={artifact.Name}>
+          <a href={artifact.URL}>{artifact.Name}</a> - {artifact.Description}
```

Comment:
From the picture, it looks like no link formatting?

What do you think are pro and con of list in which items have two parts versus 2 column table?

Reply:
> From the picture, it looks like no link formatting?

Yes. I think that is from tailwind and patternFly but my css inspection skills are lacking.

> What do you think are pro and con of list in which items have two parts versus 2 column table?

I have no preference.